### PR TITLE
feat: Add meta tag option

### PR DESCRIPTION
**Pull request by jantimon**
See #419

### CODE REVIEW

**Comment by jantimon** [comment]
@SimenB could you please take a look at this feature?

File: README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ Allowed values are as follows
 |**[`templateParameters`](#)**|`{Boolean\|Object\|Function}`|``| Allows to overwrite the parameters used in the template |
 |**[`inject`](#)**|`{Boolean\|String}`|`true`|`true \|\| 'head' \|\| 'body' \|\| false` Inject all assets into the given `template` or `templateContent`. When passing `true` or `'body'` all javascript resources will be placed at the bottom of the body element. `'head'` will place the scripts in the head element|
 |**[`favicon`](#)**|`{String}`|``|Adds the given favicon path to the output HTML|
+|**[`meta`](#)**|`{Object}`|`{}`|Allows to inject `meta`-tags. E.g. `meta: {viewport: 'width=device-width, initial-scale=1, shrink-to-fit=no'}`|
```
Comment (SimenB) [inline README.md]: it should have the example of the `value` being in object form as well, not just string

**Comment by SimenB** [comment]
This looks great to me 🙂

File: README.md (hunk 1 above)
Comment (SimenB) [reply README.md]: not necessarily here, but somewhere in the readme

Comment (jantimon) [reply README.md]: Good point

**Pull request merged**